qstrader/statistics/tearsheet.py

```diff
@@ -314,4 +314,7 @@ def plot_results(self, filename=None):
         # self._plot_txt_time(stats, ax=ax_txt_time)
 
         # Plot the figure
-        plt.show()
+        if filename:
+            plt.savefig(filename)
+        else:
```

Comment:
Wouldn't it still make sense to show the plot? I think it's a great idea to save it if a filename is included, but still think the existing behavior of showing it makes sense to remain the same